x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/series_editor.tsx

```diff
@@ -86,6 +86,10 @@ export const SeriesEditor = React.memo(function () {
     {}
   );
 
+  useEffect(() => {
+    setItemIdToExpandedRowMap({});
+  }, [allSeries.length]);
```

Comment:
i just realized this means, if a series is expanded and we delete another series, it will also collapse all the series, so this may not work.

Reply:
Ok, wasn't sure if that mattered. I will smooth that out.